File: flightReservationServer/Model/src/main/java/model/Agency.java
package model;

import java.util.Objects;

public class Agency {

    private String username;
    private String password;

    public Agency(String username, String password) {
        this.username = username;
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }


    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Agency agency = (Agency) o;
        return  Objects.equals(username, agency.username) &&
                Objects.equals(password, agency.password);
    }

    @Override
    public int hashCode() {
        return Objects.hash(username, password);
    }
}

File: FlightReservationClient/Model/Agency.cs
using System.Collections.Generic;

namespace FlightReservations.model
{
    public class Agency
    {
        public string Username;
        public string Password;

        public Agency(string username, string password)
        {
            Username = username;
            Password = password;
        }

        public override bool Equals(object obj)
        {
            var agency = obj as Agency;
            return agency != null &&
                   Username == agency.Username &&
                   Password == agency.Password;
        }

        public override int GetHashCode()
        {
            var hashCode = 568732665;
            hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(Username);
            hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(Password);
            return hashCode;
        }
    }
}
